File: Children-Competition-C#/Domain/Child.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Concurs.Domain
{
    internal class Child : Entity<Guid>
    {
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public int Age { get; set; }

        public Child(string fn, string ln, int a)
        {
            this.Id = Guid.NewGuid();
            this.FirstName = fn;
            this.LastName = ln;
            this.Age = a;
        }
        public Child(Guid guid, string fn, string ln, int a)
        {
            this.Id = guid;
            this.FirstName = fn;
            this.LastName = ln;
            this.Age = a;
        }

    }
}

File: Children-Competition-Java/mpp-proiect-java-ABNegrea/Problema5/src/main/java/Concurs/Domain/Child.java
package Concurs.Domain;

import java.util.UUID;

public class Child extends Entity<UUID> {
    private String firstName;
    private String lastName;
    private int age;

    public Child(String firstName, String lastName, int age) {
        this.setId(UUID.randomUUID());
        this.firstName = firstName;
        this.lastName = lastName;
        this.age = age;
    }

    public Child(UUID id, String firstName, String lastName, int age) {
        this.setId(id);
        this.firstName = firstName;
        this.lastName = lastName;
        this.age = age;
    }

    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public int getAge() {
        return age;
    }

    public void setAge(int age) {
        this.age = age;
    }
}
